src/core/thread/osthread.d

```diff
@@ -1941,52 +1939,41 @@ extern (C) void thread_init() @nogc
     }
     else version (Posix)
     {
-        if ( suspendSignalNumber == 0 )
-        {
-            suspendSignalNumber = SIGUSR1;
-        }
-
-        if ( resumeSignalNumber == 0 )
-        {
-            resumeSignalNumber = SIGUSR2;
-        }
-
         int         status;
-        sigaction_t sigusr1 = void;
-        sigaction_t sigusr2 = void;
+        sigaction_t suspend = void;
+        sigaction_t resume = void;
 
         // This is a quick way to zero-initialize the structs without using
         // memset or creating a link dependency on their static initializer.
-        (cast(byte*) &sigusr1)[0 .. sigaction_t.sizeof] = 0;
-        (cast(byte*) &sigusr2)[0 .. sigaction_t.sizeof] = 0;
+        (cast(byte*) &suspend)[0 .. sigaction_t.sizeof] = 0;
+        (cast(byte*)  &resume)[0 .. sigaction_t.sizeof] = 0;
 
         // NOTE: SA_RESTART indicates that system calls should restart if they
         //       are interrupted by a signal, but this is not available on all
         //       Posix systems, even those that support multithreading.
         static if ( __traits( compiles, SA_RESTART ) )
-            sigusr1.sa_flags = SA_RESTART;
-        else
-            sigusr1.sa_flags   = 0;
```

Comment:
Removed this because it's already `0` thanks to the initialization at L1960 / 1948